src/main/java/com/online/bookstore/dto/cart/ShoppingCartDto.java

```diff
@@ -1,10 +1,10 @@
 package com.online.bookstore.dto.cart;
 
-import java.util.Set;
+import java.util.List;
 
 public record ShoppingCartDto(
         Long id,
         Long userId,
-        Set<CartItemDto> cartItems
+        List<CartItemDto> cartItems
```

Comment:
Please, explain the reason for that :)

Reply:
ordering, by default set places items by hash code, it looks prettier if we have ordered cart items